packages/core-data/src/entities.js

```diff
@@ -46,8 +41,8 @@ async function loadPostTypeEntities() {
  *
  * @return {Promise} Entities promise
  */
-async function loadTaxonomyEntities() {
-	const taxonomies = await apiFetch( { path: '/wp/v2/taxonomies?context=edit' } );
+function * loadTaxonomyEntities() {
```

Comment:
Looks like we need an eslint rule for `function*` over `function *`, because this file is the only place you wrote it `function *` and it looks weird to me 😄